.dockerignore

```diff
@@ -0,0 +1 @@
+.venv
```

Comment:
Consider adding more entries to the `.dockerignore` file, such as `.git` and `.DS_Store`, to exclude unnecessary files and directories from the Docker image.